src/Microsoft.Health.Dicom.Api/Microsoft.Health.Dicom.Api.csproj

```diff
@@ -13,6 +13,14 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.Health.Dicom.Core\Microsoft.Health.Dicom.Core.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Dicom.Core">
```

Comment:
Do we need a PR for this? Looks like just local testing?

Reply:
It's just for local testing. The PR is sharing the result to Abdullah.
I think github should exclude draft PR from homepage :(